src/request.py

```diff
@@ -16,11 +20,16 @@ def request(method_type: str, url: str, **kwargs):
                               data=data,
                               auth=kwargs.get("auth", None))
     if result.status_code != 200:
+        log.exception(f'request status code !=200 ({result.status_code})')
         raise ValueError("oops ", result.text)
+
     try:
         response_tag = ElementTree.fromstring(result.text)
+        log.debug(f'trying to parse as xml')
         if response_tag.attrib["stat"] == 'fail':
+            log.exception(f'server returned fail status in xml')
             raise ValueError(result.text)
+        log.debug(f'trying parsed as xml')
     except ElementTree.ParseError:
-        pass
+        log.debug(f'data received')
```

Comment:
Is this a correct message?

Reply:
changed to 'request returned correct response'